### ISSUE TITLE
[minor bug] wrong default selection color in RichTextBox

### ISSUE THREAD

**Issue by theuserbl** [dotnet/winforms#2689]
![WinFormsz](https://user-images.githubusercontent.com/8337032/72207205-06fa6280-3497-11ea-8651-5fd5f28de234.png)

On the left side is with .net Framework v4.0.30319 on Win8.1 and on the right side is .net Core 3.1.

And so, on the .net Framework selected text is in a ListBox and the RichTextBox white with blue background. But on .net Core 3.1 the selected RichTextBox text ist black.

The program was created with the designer of  Visual Studio 2017.

Because in my last bug-report at https://github.com/dotnet/winforms/issues/2685 weltkante said, that the default font changed from .net Framework to .net Core, I have set the font to the same, like in his posted link.

**Comment by Zheng-Li01** [comment]
@theuserbl, the .NET Core 3.1 behavior maybe keep up with the latest .NET Framework version. The default section color in RichTextBox is white with blue background when the .NET Framework version <= 4.6.2; and the default section text's color in RichTextBox is black when the  .NET Framework version >= 4.7. See below screenshot.
![Update](https://user-images.githubusercontent.com/38325459/72230964-b3e9f200-35f3-11ea-9d14-d9226c96b399.PNG)

@merriemcgaw, @Tanya-Solyanik, is it expected that the default section color is different with the different .NET Framework version for the RichTextBox.

**Comment by RussKie** [comment]
@M-Lipin @vladimir-krestov does it have anything to do with a11y requirements?

**Comment by merriemcgaw** [comment]
@Zheng-Li01 can you ensure the contrast of selected text is OK with the black text, light blue selection? If so then we're good to close this issue. But I didn't want to lose it and have this be inaccessible.

**Comment by RussKie** [comment]
@merriemcgaw I suspect this may be somewhat related to the overall theming discussions, see https://github.com/dotnet/winforms/issues/2543

**Comment by Tanya-Solyanik** [comment]
@Zheng-Li01 , if contrast ratio is too low, could you please also test the 4.7 Framework app with 
"Switch.System.Windows.Forms.DoNotLoadLatestRichEditControl" set to true? This will opt out from the using RichEdit version 5. In Core, we removed all quirks and now Core always loads the latest rich edit control version. I suspect that selection color comes from the native control, rather than winforms.

**Comment by Zheng-Li01** [comment]
@merriemcgaw, the color contrast for the black text, light blue selection is ok, see below screenshot.
![GithubIssue](https://user-images.githubusercontent.com/38325459/74899048-bac80980-53d6-11ea-8c7a-a6bcdf380f09.PNG)
@Tanya-Solyanik, testing the winforms app targeting .NET >= 4.7, after setting the"Switch.System.Windows.Forms.DoNotLoadLatestRichEditControl" set to true, the app using the RichEdit version 5. see below screenshot.
![GithubIssue2](https://user-images.githubusercontent.com/38325459/74899441-ed263680-53d7-11ea-8ed8-3aa7d6dcf6a9.PNG)

**Comment by M-Lipin** [comment]
So in the context of Accessibility requirements (contrast ratio) there is no violation. But in the context of consistency it should be fixed. @merriemcgaw, is it correct?

**Comment by Tanya-Solyanik** [comment]
I don't see highlight color exposed in RichEdit [APIs](https://docs.microsoft.com/en-us/windows/win32/controls/about-rich-edit-controls)

**Comment by RussKie** [comment]
This is how the Notepad has it:
![image](https://user-images.githubusercontent.com/4403806/74989817-98e18c00-5495-11ea-8b1f-6d105387c3cc.png)

So I guess we may want it fixed

**Comment by merriemcgaw** [comment]
I'm going to remove the accessibility label since @M-Lipin has shown there's no issue there. As far as whether we fix it - if it's not in the APIs as @Tanya-Solyanik showed then _can_ we fix it?

**Comment by Tanya-Solyanik** [comment]
>then can we fix it? 

Maybe office quirked this color? I would think this is breaking for native applications that host rich edit and win32 contorls as well. But I would say, this is a low pri issue.

notepad is a win32 EDIT control:
![image](https://user-images.githubusercontent.com/15823268/75212424-c0f21900-573b-11ea-8fb5-bbfc471e2c8e.png)
WinWord is the richedit - 
![image](https://user-images.githubusercontent.com/15823268/75212455-d9623380-573b-11ea-82a5-9dd57b93cb84.png)

These are 2 different apps, they don't have to match

**Comment by merriemcgaw** [comment]
I'll leave this marked as future and if we find some time we can investigate. But you're right, I highly doubt this is a blocking issue.
